converters/fromZigbee.js

```diff
@@ -1643,20 +1643,62 @@ const converters = {
                     default:
                         // if (meta.logger) meta.logger.debug(`plug.mmeu01: unknown vtype=${data[i+1]}, pos=${i+1}`);
                     }
-                    payload[index] = value;
-                    // if (meta.logger) meta.logger.debug(`plug.mmeu01: recorded index ${index} with value ${value}`);
+                    switch (index) {
```

Comment:
I think we should compact this code with if else. e.g.
```js
if (index === 3) payload.temperature = calibrateAndPrecisionRoundOptions(value, options, 'temperature');
else if (index === 100) payload.state = value === 1 ? 'ON' : 'OFF';
etc...
```

Sames goes for all if's here: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1874/files#diff-b676ab0df4d49b8572967cb4c92a2dc6264091131f6b1828ca13d0f9656e367aR1677

Reply:
Not see a big difference, but - as you wish ;-)